zipkin-server/src/main/java/zipkin2/server/internal/elasticsearch/ZipkinElasticsearchStorageProperties.java

```diff
@@ -35,6 +35,63 @@
     BODY
   }
 
+  public static class Ssl {
+    private String keyStore = emptyToNull(System.getProperty("javax.net.ssl.keyStore"));
```

Comment:
Assuming this works, I'll just put these java properties in an example in the server README as they are shorter than spelling out the spring boot variants.